db/dcrpg/sync.go

```diff
@@ -201,7 +201,7 @@ func (db *ChainDB) SyncChainDB(ctx context.Context, client rpcutils.MasterBlockG
 	}
 
 	// Safely send new block hash on updateExplorer channel, and set the channel
-	// to nil if it was closed or the buffer is full.
+	// to nil if the buffer is full.
```

Comment:
`(*WiredDB).resyncDB` has similar incorrect language.  Would you update there too please?